<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Verificar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cuadro_0">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cuadro_1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cuadro_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cuadro_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cuadro_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cuadro_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cuadro_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cuadro_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cuadro_8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>407</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>verificar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sumar()</slot>
  <slot>verificar()</slot>
 </slots>
</ui>
